### PR TITLE
Nisp 2018

### PR DESCRIPTION
**Pull request by prashantchoudhari**

### CODE REVIEW

File: app/uk/gov/hmrc/nisp/views/excluded_mci.scala.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        <div class="section-border"><h2 class="heading-medium">DECISION TO BE MADE.    How to contact us</h2></div>
+        <ul class="list-bullet">
+            <li>Telephone: 0300 200 3300</li>
+            <li>Textphone: 0300 200 3319</li>
+            <li>Outside UK: +44 135 535 9022</li>
+        </ul>
+        <p>Phone lines are open 8am to 8pm Monday to Friday and 8am to 4pm on Saturday.</p>
+        <p>Closed Sundays and bank holidays.</p>
+        <p>Phone lines are less busy before 10am Monday to Friday.</p>
+        <p><a href="https://www.gov.uk/call-charges" rel="external" target="_blank" data-journey-click="checkyourstatepension:external:callcharges">Find out about call charges (opens in a new window)</a></p>
```
Comment (dspork) [inline app/uk/gov/hmrc/nisp/views/excluded_mci.scala.html]: All of the messaging needs to be in the messages files.

File: app/uk/gov/hmrc/nisp/views/excluded_sp.scala.html [hunk 2]
```diff
@@ -50,22 +50,26 @@ <h2 class="heading-medium"> @Html(Messages("nisp.excluded.willReach", Dates.form
         <div class="panel-indent">
             @if(spExclusions.contains(CustomerTooOld) || spExclusions.contains(PostStatePensionAge)) {
                     @exclusions.sp.postStatePensionAge()
-            } else { 
-
-                @if(spExclusions.contains(AmountDissonance)) {
-                    @exclusions.sp.amountDissonance()
+            } else {
+                @if(spExclusions.contains(ManualCorrespondenceIndicator)) {
+                    @exclusions.sp.isleOfMan()
```
Comment (dspork) [inline app/uk/gov/hmrc/nisp/views/excluded_sp.scala.html]: I don't think we need an MCI if statement here. And it shouldn't go to the IOM page...

File: app/uk/gov/hmrc/nisp/views/excluded_sp_old.scala.html [hunk 3]
```diff
@@ -37,10 +37,14 @@
     @main(browserTitle=Some(Messages("nisp.main.title")), userLoggedIn = true,
         pageTitle = Some(Messages("nisp.excluded.title")), analyticsAdditionalJs = Some(analyticsAdditionalJs)) {
 
+        @if(spExclusions.exclusions.contains(Exclusion.ManualCorrespondenceIndicator)) {
+            @exclusions.dead()
+            ???
```
Comment (dspork) [inline app/uk/gov/hmrc/nisp/views/excluded_sp_old.scala.html]: Again I don't think we need an MCI if statement, and it shouldn't go the dead exclusion or produce ???

Comment (roundrobin99) [inline app/uk/gov/hmrc/nisp/views/excluded_sp_old.scala.html]: This was my original code when I was trying to fit the logic. I wanted to display an error message so I could test it, but I never got that far,

Also the `???` means I don't know what to write next

File: app/uk/gov/hmrc/nisp/views/excluded_sp.scala.html (hunk 2 above)
Comment (roundrobin99) [inline app/uk/gov/hmrc/nisp/views/excluded_sp.scala.html]: This was my first stab at this and wanted to display "a" error message for testing purposes. Obviously it should display the MCI error page.

**Comment by prashantchoudhari** [comment]
@InduTest - this is ready to be merged once you complete testing.

my changes are reviewed by @dspork and his latest changes for hiding breadcrumb and adding age are also good to go.

**Pull request merged**